app/payment-request/get-payment-request-matching-reference.js

```diff
@@ -0,0 +1,20 @@
+const { CS } = require('../constants/schemes')
+const db = require('../data')
```

Comment:
Swap these two around

Reply:
I'm thinking the db reference should be first? As it's calling something external, unless I'm mistaken?

Reply:
Don't think it really matters but have swapped :)